src/components/chips/js/contactChipsAutocompleteScope.js

```diff
@@ -0,0 +1,29 @@
+angular
+    .module('material.components.chips')
+    .directive('mdContactChipsAutocompleteScope', MdContactChipsAutocompleteScope);
+
+function MdContactChipsAutocompleteScope ($compile, $mdUtil) {
+  return {
+    restrict: 'A',
+    link:     postLink,
+    scope:    false
+  };
+
+  function postLink (scope, element, attr) {
+    // Since the autocomplete popup is appended to the body, we cannot traverse up the DOM looking
+    // for the proper controller and grabbing it's scope, so we must go WAY up to get the proper
+    // scope; create a new child scope based off of the parent, and copy the item over on changes
+    var newScope = scope.$parent.$parent.$parent.$new();
```

Comment:
If not, I'd maybe pass the scope into this directive.  In either case, what you have now is fine, I just try to avoid the $parent chain whenever possible.